framework/basehandlers.py

```diff
@@ -30,6 +30,7 @@
 from framework import permissions
 from framework import ramcache
 from framework import xsrf
+from framework import secrets
```

Comment:
nit: move up one line to keep alphabetical?

Reply:
Done